src/app/lib/config/services/hausa.js

```diff
@@ -245,12 +245,12 @@ export const service = {
         url: '/hausa/wasanni',
       },
       {
-        title: 'Mujalla',
-        url: '/hausa/mujalla',
+        title: 'Nishadi',
+        url: '/hausa/topics/cg726kz37wdt',
```

Comment:
This topic link will work on live `www.bbc.com/hausa/topics/cg726kz37wdt` but because of the way that Topic page IDs are generated, it will not work on test. e.g. the following will 404: `www.test.bbc.com/hausa/topics/cg726kz37wdt`. There is a longer id which is called the *LDP tag id* which is identical over Test and Live and we should use this for the meantime until a longer-term solution is discussed between Product and Engineering in our team and in the Topic CMS team. 

```suggestion
        url: '/hausa/topics/1c3b60a9-14eb-484b-a750-9f5b1aeaac31',
```